src/System.Device.Gpio/System/Device/Pwm/Channels/UnixPwmChannel.Linux.cs

```diff
@@ -38,6 +40,11 @@ internal class UnixPwmChannel : PwmChannel
             _channel = channel;
             Validate();
             Open();
+
+            // avoid opening the file for operations changing relatively frequently
+            _dutyCycleWriter = new StreamWriter(new FileStream($"{_channelPath}/duty_cycle", FileMode.Open, FileAccess.ReadWrite));
```

Comment:
Does the fileStream need to be disposed as well or does it get disposed by the streamwriter?

Reply:
streamwriter disposes it